crates/cervo-runtime/src/runtime.rs

```diff
@@ -42,6 +43,25 @@ impl Runtime {
         }
     }
 
+    pub fn queue_len(&self) -> usize {
+        self.queue.len()
+    }
+
+    pub fn model_len(&self) -> usize {
+        self.models.len()
+    }
+
+    pub fn brain_ids(&self) -> Vec<BrainId> {
+        self.models.iter().map(|model| model.id).collect()
+    }
+
+    pub fn all_input_shapes(&self) -> Vec<Vec<(String, Vec<usize>)>> {
+        self.models
+            .iter()
+            .map(|model| model.inferer.input_shapes().to_vec())
+            .collect()
+    }
```

Comment:
All these functions seem a bit weird. I'd like all of them to be removed if possible.

Reply:
Agreed. This would require me to make models public, and remove (crate) from `ModelState`, if that is okay

Reply:
Noooo :D Why?